security/ima-evm-utils-tests.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2022 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import build, distro, git, process
+from avocado.utils.software_manager.manager import SoftwareManager
+
+
+class IMAEVMUtils(Test):
+    """
+    ima-evm-utils testsuite
+    :avocado: tags=security,testsuite
```

Comment:
@nasastry one last suggestion,, just one or 2 lines to explain what does the test do ?

Reply:
@nasastry you can have a Readme.txt for more data points